CorsixTH/Src/th_movie.h

```diff
@@ -321,70 +321,71 @@ class THMovie
     //! \param pFrame An empty frame which gets populated by the data in the
     //! packet queue.
     //! \returns FFMPEG result of avcodec_recieve_frame
-    int getFrame(int stream, AVFrame* pFrame);
+    int get_frame(int stream, AVFrame* pFrame);
 #else
     //! Convert video packet data into a frame.
     //!
     //! \param pFrame An empty frame which gets populated by the data in the
     //! video packet queue.
     //! \returns 1 if the frame was received, 0 if it was not, and < 0 on error
-    int getVideoFrame(AVFrame *pFrame);
+    int get_video_frame(AVFrame *pFrame);
 #endif
 
-    SDL_Renderer *m_pRenderer; ///< The renderer to draw to
+    SDL_Renderer *renderer; ///< The renderer to draw to
```

Comment:
th_gfx_font has code like `render_target* pCanvas`, while here the `*` is near the variable.
I prefer the latter, as it matches the true semantics of the pointer declaration, but at least it's inconsistent.

Reply:
I will be switching all of these to C++ style in the format PR.

https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#nl18-use-c-style-declarator-layout

http://www.stroustrup.com/bs_faq2.html#whitespace